### ISSUE TITLE
Add support for all of the unlicensed mapper types in GBX

### ISSUE THREAD

**Issue by endrift** [mgba-emu/mgba#2437]
At time of writing, the following mapper types from the [GBX spec](http://hhug.me/gbx/mappers) are not supported

Targeting 0.10:
- [ ] Sintax
- [ ] NT older type 1
- [ ] NT older type 2
- [ ] Sachen MMC1
- [ ] Sachen MMC2

Targeting 0.11:
- [ ] M161
- [ ] Li Cheng
- [ ] "Last Bible" multicart
- [ ] Liebao Technology
- [ ] Rocket Games/Datel
- [ ] "New GB Color" HK
- [ ] Kongfeng GGB81

GBX support is a separate issue.

**Comment by ReNardier** [comment]
don't forget about TPP1, it's under homebrew on the GBX spec
https://github.com/aaaaaa123456789/tpp1/blob/master/specification.md

**Comment by endrift** [comment]
I don't believe TPP1 was in the spec yet when I first made this list. I think I'm the one who mentioned it to taizou in the first place though.

**Comment by ReNardier** [comment]
good to know
